### PR TITLE
Fix lifestealer infest

### PR DESCRIPTION
**Pull request by JoshdanG**
1. Looks like infest moved from ability number 6 to number 5.  This fixes code to auto-adapt that was not working properly
2. Changes to comments:
     - Added comments to abilities
     - Removed commented out code
     - Removed comment that infest would be used to stun

### CODE REVIEW

File: ability_item_usage_life_stealer.lua [hunk 1]
```diff
@@ -457,23 +456,23 @@ local consumeThink = function()
...
 end
 
-Consider[5] = function()
-	if AbilitiesReal[5]:GetName() == "life_stealer_consume" then
-		return consumeThink()
-	end
-	if AbilitiesReal[5]:GetName() == "life_stealer_infest" then
-		return infestThink()
+-- Some previous code seems to hint these can switch places.
+-- This might actually be unnecessary
```
Comment (adamqqqplay) [inline ability_item_usage_life_stealer.lua:460]: The ability position change here is very interesting, it seems to be a bug in Dota2 itself.

**Review by adamqqqplay — APPROVED** [review]
Others looks good to me. Thanks for your fix.

**Review by ArcherWayne — APPROVED** [review]
Tested and seems worked fine.

File: ability_item_usage_life_stealer.lua (hunk 1 above)
Comment (JoshdanG) [reply ability_item_usage_life_stealer.lua:460]: Not sure if you'd like me to change something here (maybe the comment?).  If these two abilities do swap places for whatever reason, this updated code should handle it better than the previous code.

Comment (adamqqqplay) [reply ability_item_usage_life_stealer.lua:460]: > Not sure if you'd like me to change something here (maybe the comment?). If these two abilities do swap places for whatever reason, this updated code should handle it better than the previous code.

No changes needed, I'm just curious about this kind of bug.

**Pull request merged**